salt/pillar/nacl.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+'''
+Decrypt pillar data through the builtin NACL renderer
+
+In most cases, you'll want to make this the last external pillar used. For
+example, to pair with the builtin stack pillar you could do something like
+this:
+
+.. code:: yaml
+
+    nacl.config:
+        keyfile: /root/.nacl
+
+    ext_pillar:
+      - stack: /path/to/stack.cfg
+      - nacl: {}
+
+Set ``nacl.config`` in your config.
+
+'''
+
+from __future__ import absolute_import
+import salt.utils
```

Comment:
Why is this import necessary? I'm not seeing it used anywhere. I'm guessing this was copied from the GPG rendered but tbh I'm not sure I follow the logic there either.

Reply:
@epcim Did you see the above question?

Reply:
salt.utils is quite sure not needed, it's copy/paste from other recently added ext pillar processors.